create-hpc-cluster/azuredeploy.json

```diff
@@ -121,25 +120,24 @@
         "virtualNetworkName": "[concat(parameters('clusterName'), 'VNet')]",
         "addressPrefix": "10.0.0.0/16",
         "subnet1Name": "Subnet-1",
-        "subnet1Prefix":"10.0.0.0/24",
+        "subnet1Prefix": "10.0.0.0/24",
         "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
         "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
-        "publicIPAddressType":  "Dynamic",
+        "publicIPAddressType": "Dynamic",
         "publicIPAddressName": "[concat(parameters('clusterName'), '-PublicIP-VM')]",
         "publicIPAddressNameHN": "[concat(variables('publicIPAddressName'),'-HN')]",
         "dnsName": "[concat(parameters('clusterName'), 'ep')]",
         "nicName": "[concat(parameters('clusterName'),'-nic')]",
         "nicNameHN": "[concat(variables('nicName'),'-hn')]",
-        "vmExtensionName": "customScriptExtension",
         "availableCNImageSkus": {
             "ComputeNode": "2012R2CN",
             "ComputeNodeWithExcel": "2012R2CNExcel"
         },
         "computeNodeImageSku": "[variables('availableCNImageSkus')[parameters('computeNodeImage')]]",
```

Comment:
variables('availableCNImageSkus') is the dictionary, and parameters('computeNodeImage') is the key. I am using dictionary[key] to get the value.